flex/exceptions.py

```diff
@@ -103,3 +103,6 @@ class NoParameterFound(ValueError):
 
 class MultipleParametersFound(ValueError):
     pass
+
+class DataCouldNotBeParsed(ValueError):
+    pass
```

Comment:
> I'm not sure this is the best approach, it means that either `json.JSONDecodeError` or `flex.http.JSONDecodeError` could be raised

I believe all of the following should work with both python 2 and 3 with the scheme that I proposed.

```python
from flex.exceptions import JSONDecodeError

try:
    # some code
except JSONDecodeError:
    # handle things

from json import JSONDecodeError  # will only work in python3

try:
    # some code
except JSONDecodeError:
    # handle things

# vanilla ValueError based handling

try:
    # some code
except ValueError:
    # handle things
```

This seems like an acceptable API for users to catch this.  thoughts?